assets/constructor_macOS.patch

```diff
@@ -62,8 +62,6 @@ diff -Naur constructor-orig/osxpkg.py constructor-patched/osxpkg.py
 +        options.set('customize', 'never')
 +        options.set('customLocation', '/Applications/MNE-Python')
 +        options.set('rootVolumeOnly', 'true')
```

Comment:
Ohh we still want this stuff so we get rid of the entire patch, just parts of it. Makes sense

Reply:
Yes we want to keep that :)